gamestonk_terminal/stocks/fundamental_analysis/yahoo_finance_view.py

```diff
@@ -165,6 +162,31 @@ def display_dividends(ticker: str, num: int = 12, export: str = ""):
         return
     div_history["Dif"] = div_history.diff()
     div_history = div_history[::-1]
+    if plot:
+        fig, ax = plt.subplots(
+            figsize=plot_autoscale(), constrained_layout=False, dpi=PLOT_DPI
+        )
+        ax.plot(
+            div_history.index,
+            div_history["Dividends"],
+            ls="-",
+            linewidth=0.75,
+            marker=".",
+            markersize=4,
+            mfc="k",
+            mec="k",
+            c="k",
+            alpha=1,
+        )
+        ax.set_xlabel("Date")
+        ax.set_ylabel("Amount ($)")
+        ax.set_title(f"Dividend History for {ticker}")
+        ax.set_xlim(div_history.index[-1], div_history.index[0])
+        if gtff.USE_ION:
+            plt.ion()
+        fig.tight_layout()
+        plt.show()
+
```

Comment:
Basically the `else:` should come on this line.

You should have something like:
```
if plot:
   do something for plotting
else:
   just show the raw data
```

Reply:
@deeleeramone can you have a look at this so we can merge it?